lib/workers/branch/index.spec.ts

```diff
@@ -1044,6 +1051,12 @@ describe('workers/branch/index', () => {
 
       fs.outputFile.mockReturnValue();
       fs.readFile.mockResolvedValueOnce(Buffer.from('modified file content'));
```

Comment:
Are you sure `fs-extra` is called now, you mocked `util/fs` and all code should only use that instead of directly using `fs-extra`.

So i think all those can now be removed as they are no longer used by actual code?